java/bench/src/java/org/apache/orc/bench/CompressionKind.java

```diff
@@ -53,6 +54,8 @@ public OutputStream create(OutputStream out) throws IOException {
         return new GZIPOutputStream(out);
       case SNAPPY:
         return new SnappyCodec().createOutputStream(out);
+      case ISAL:
```

Comment:
1. igzip is only part of the ISAL.
2. igzip only support level 0 - level 1 compression, it can not replace libz totally.
3. igzip's API can not replace ligz's API directly.